bridge/slack/slack.go

```diff
@@ -245,6 +248,13 @@ func (b *Bslack) sendWebhook(msg config.Message) (string, error) {
 }
 
 func (b *Bslack) sendRTM(msg config.Message) (string, error) {
+	if msg.Event == config.EVENT_USER_TYPING {
+		if b.GetBool("ShowUserTyping") {
+			chanID := b.channelsByName[msg.Channel].ID
```

Comment:
Don't read the map directly, locking issues. Use `getChannelByName()` function

Reply:
heh just realized we fetched this right below